### PR TITLE
added card, header

### PR DESCRIPTION
**Pull request by sabrinababakulova**
создать приложение, у приложения должна быть шапка и должен быть блок (далее “Карточка”) с заголовком и текстом. Текст может быть любым. Шапка и Карточка должны быть стилизированы (подключить любую библиотеку компонентов типа материала).

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
```
Comment (vshynkarova) [inline public/manifest.json]: Update default icon to some custom. Update defauld name.

File: src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
```
Comment (vshynkarova) [inline src/components/Card.jsx]: Create folder for this component with 'index' file

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
```
Comment (vshynkarova) [inline src/components/Header.jsx]: Create sepate folder for this component with 'index' file

File: src/theme.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+export const theme = {
```
Comment (vshynkarova) [inline src/theme.js]: Read this and try to use some default values for all you components. https://chakra-ui.com/docs/styled-system/theming/theme?scroll=true

**Review by vshynkarova — COMMENTED** [review]
Additional tasks:
1. For this and future pull requests: Add a screenshot of your UI changes to the description.
2. Create pull request template  for your repository (add such items like: description, screnshots). You can add more elements to the template of your choice.

**Comment by sabrinababakulova** [comment]
Changed UI based on the style props that were provided by Chakra
changes to:
Favicon, Card, Header
image

changed manifest.json

**Review by vshynkarova — APPROVED** [review]

**Pull request merged**